packages/platform/atoms/vite.config.ts

```diff
@@ -28,7 +28,7 @@ export default defineConfig({
       fileName: "cal-atoms",
     },
     rollupOptions: {
-      external: ["react", "fs", "path", "os", "react-dom"],
+      external: ["react", "fs", "path", "os", "react-dom", "crypto"],
```

Comment:
do we still need to exclude crypto if atoms is not importing libraries ?

Reply:
Good point - removed `crypto` and still can build atoms. Removed `crypto` in [refactor: remove crypto from atoms vite rollupOptions](https://github.com/calcom/cal.com/pull/14870/commits/c563676d9be4ed5c6b4d5e14cc1d8146bd47f40a)